lib/presentation/redux/reducers/app_reducer.dart

```diff
@@ -47,6 +48,7 @@ import 'package:linshare_flutter_app/presentation/redux/states/network_connectiv
 
 AppState appStateReducer(AppState state, action) {
   if (state.networkConnectivityState.connectivityResult == ConnectivityResult.none &&
+      action is ActionOnline &&
       !(action is SetNetworkConnectivityStateAction) &&
       !(action is CleanNetworkConnectivityStateAction)) {
```

Comment:
this is hard to understand, please separate in a method

Reply:
Done, pls check again

Reply:
more clearly, thank you